packages/plugins/datasource/src/js/datasource.js

```diff
@@ -15,7 +15,7 @@ import { useResource } from '@opentiny/tiny-engine-controller'
 import { generateFunction } from '@opentiny/tiny-engine-controller/utils'
 import { isMock } from '@opentiny/tiny-engine-common/js/environments'
 import { utils as commonUtils, constants } from '@opentiny/tiny-engine-utils'
-import { read, utils, writeFileXLSX } from 'xlsx'
+import { read, utils, writeFileXLSX } from 'xlsx/xlsx.mjs'
```

Comment:
In my opinion, the following two lines of code are equivalent

```
import { read, utils, writeFileXLSX } from 'xlsx/xlsx.mjs'
import { read, utils, writeFileXLSX } from 'xlsx'
```

What is the difference between them?